packages/react-ui-core/src/Button/Button.js

```diff
@@ -31,12 +31,14 @@ export default class Button extends Component {
     return (
       <button
         {...props}
-        className={classNames(
-          className,
-          theme.Button,
-          theme[`Button-${color}`],
-          theme[`Button-${size}`],
-        )}
+        className={
+          classNames(
```

Comment:
did you want to revise this to `classnames` as well?

Reply:
Doh! yep...